crud/src/main/java/org/openmbee/sdvc/crud/controllers/orgs/OrgsController.java

```diff
@@ -59,12 +61,14 @@ public OrgsController(OrganizationRepository organizationRepository) {
     public ResponseEntity<? extends BaseResponse> handlePost(
         @RequestBody OrganizationsRequest orgPost) {
 
+        OrganizationsResponse response = new OrganizationsResponse();
         if (!orgPost.getOrgs().isEmpty()) {
             logger.info(orgPost.getOrgs().get(0).getId());
-            OrganizationsResponse response = new OrganizationsResponse();
 
             for (OrgJson org : orgPost.getOrgs()) {
-                //TODO reject if orgId isn't there
+                if (org.getId() == null || org.getId().isEmpty()) {
+                    throw new BadRequestException(response.addMessage("Organization ID not found"));
```

Comment:
It's just that this is a check for an ID, so without that, it would just say the error message as many times as there are invalid org objects. I'll add the messages into the response though.